configuration/appointmentservicetypes/appointmentservicetypes.csv

```diff
@@ -0,0 +1,13 @@
+Uuid,Void/Retire,Name,Service definition,Duration
+9afa9a6b-54bc-49af-9be7-6c62d7be1436,,"Follow up ",New Patient,
```

Comment:
Rather than having 'New Patient' here: it should be all the other apptservice definitions
Compare how serviceDefinitions from [here](https://github.com/MSF-OCB/openmrs-config-maadi/blob/main/configuration/appointmentservicedefinitions/servicedefinitions.csv) were added [here](https://github.com/MSF-OCB/openmrs-config-maadi/blob/main/configuration/appointmentservicetypes/servicetypes.csv)